prototype_source/pt2e_quant_ptq_x86_inductor.rst

```diff
@@ -165,11 +165,35 @@ After we get the quantized model, we will further lower it to the inductor backe
 
 ::
 
-    optimized_model = torch.compile(converted_model)
+    with torch.no_grad():
+        optimized_model = torch.compile(converted_model)
+
+        # Running some benchmark
+        optimized_model(*example_inputs)
 
-    # Running some benchmark
-    optimized_model(*example_inputs)
+In a more advanced scenario, int8-mixed-bf16 quantization comes into play. In this instance,
+a Convolution or GEMM operator produces BFloat16 output data type instead of Float32 in the absence
+of a subsequent quantization node. Subsequently, the BFloat16 tensor seamlessly propagates through
+subsequent pointwise operators, effectively minimizing memory usage and potentially enhancing performance.
```

Comment:
Shall we highlight the fact that the usage is as simple as wrapping the script under the bfloat16 autocast context?

Reply:
Thanks for the suggestion. Modified the description to highlight it.